css.go

```diff
@@ -427,3 +444,124 @@ func ParseAttributeSelector(selector string) (AttributeSelector, error) {
 		matchFn: match,
 	}, nil
 }
+
+type procesor func(map[string]string, string, string)
```

Comment:
nit: procesor -> processor

Reply:
nice call. I wonder if its worth while to https://github.com/marketplace/actions/github-spellcheck-action or something as a ci action?

Reply:
Yeah golangci-lint + that (unless golangci-lint has typos built in) would be good